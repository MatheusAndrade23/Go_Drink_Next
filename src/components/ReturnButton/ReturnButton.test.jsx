import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { ReturnButton } from '.';

describe('<ReturnButton />', () => {
  it('should render', () => {
    const { container } = renderTheme(<ReturnButton />);
    expect(container).toMatchSnapshot();
  });
});
