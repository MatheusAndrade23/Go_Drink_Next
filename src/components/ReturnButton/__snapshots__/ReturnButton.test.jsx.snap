// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReturnButton /> should render 1`] = `
.c0 {
  position: fixed;
  color: #00ADB5;
  top: 10px;
  left: 0px;
  height: 50px;
  width: 50px;
  font-size: 56px;
}

.c0:hover {
  color: #EEEEEE;
}

@media (max-width:768px) {
  .c0 {
    font-size: 40px;
    top: 20px;
  }
}

@media (max-width:420px) {
  .c0 {
    display: none;
  }
}

<div>
  <a
    class="c0"
    href="/"
    title="return"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 512 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M217.9 256L345 129c9.4-9.4 9.4-24.6 0-33.9-9.4-9.4-24.6-9.3-34 0L167 239c-9.1 9.1-9.3 23.7-.7 33.1L310.9 417c4.7 4.7 10.9 7 17 7s12.3-2.3 17-7c9.4-9.4 9.4-24.6 0-33.9L217.9 256z"
      />
    </svg>
  </a>
</div>
`;
